portfolio/src/main/webapp/blog.html

```diff
@@ -55,21 +56,26 @@ <h4>Software Engineering Intern</h4>
       <section class="banner">
         <div class="info">
           <h2>Blog Title</h2>
-          
           <p> Insert some blog post content here. </p>
-
+          
           <h2>All Comments</h2>
-          <div id="comment-container"></div>
+          <div id="comment-container">
+            <ul id="comment-list"></ul>
+          </div>
 
-          <h2>Leave A Comment</h2>
+          <h2>Leave A Reply</h2>
           <form action="/data" method="POST">
-            <label for="name">Name:</label>
-            <input type="text" id="name" name="name"> 
-            <label for="feedback">Comments:</label>
-            <textarea id="feedback" name="text-input"></textarea>
-            <input type="submit">
+            <div class="inputBox">
+              <label for="name">Name</label>
+              <input type="text" id="name" name="name" required="required">
+            </div>
+            <div class="inputBox">
+              <label for="feedback">Comment</label>
+              <textarea id="feedback" name="text-input" cols ="100" rows="8" maxlength="10000" required="required"></textarea>
```

Comment:
Just by the way, marking it as required is purely a convenience thing - it's possible for the user to bypass, so in a robust application, you'd want to validate in your server-side code as well that all necessary fields were specified.